.github/workflows/on-demand-build-docker-image-deploy-preview.yml

```
@@ -22,6 +22,17 @@ jobs:
             PR: ${{ github.event.client_payload.pull_request.number }}.
             recreate: ${{ github.event.client_payload.slash_command.args.named.recreate }}.
 
+      - name: Git checkout code
+        uses: actions/checkout@v3
+
+      - name: Fail dp incase of merge conflicts
+        run: |
+          status="$(gh pr view ${{ github.event.client_payload.pull_request.number }} --json mergeable --jq .mergeable)";
+          if [ $status = "CONFLICTING" ]; then
```

Please prefer double square brackets.

```suggestion
          if [[ $status == "CONFLICTING" ]]; then
```